openedx/core/djangoapps/video_pipeline/models.py

```diff
@@ -11,40 +11,6 @@
 import six
 
 
-class VideoPipelineIntegration(ConfigurationModel):
```

Comment:
I think we need to split this out into a separate PR to avoid errors during deployment (first PR removes any code that references this model, then second PR removes the model.

Reply:
PR updated